src/figment.rs

```diff
@@ -287,7 +287,8 @@ impl Figment {
     /// ```
     pub fn extract_inner<'a, T: Deserialize<'a>>(&self, key: &str) -> Result<T> {
         let merged = self.merged().map_err(|e| e.resolved(self))?;
-        let inner = merged.find(key).ok_or(Kind::MissingField(key.to_string().into()))?;
+        let inner = merged.find(key)
```

Comment:
Here I'm not sure if the formattnig was a good decision. Please tell me if I should correct it.